### DISCUSSION TITLE
FZF File Search with Tab

### DISCUSSION THREAD

**Discussion by jrock2004** [LazyVim/LazyVim#3657] [Q&A] [answered]
So I am trying out FZF file search vs telescope. I hit the keymap and hit tab go through the items. When I find the one I want and hit enter, it just opens the first item in the list instead of highlighted one. If I use arrow keys it works just fine. Do I need to update a config option?

**Comment by dpetka2001** [comment]
https://github.com/ibhagwan/fzf-lua/discussions/1170

**Comment by iamxiaojianzheng** [comment]
Because the default behavior of `Tab` is to select multiple items, if you want `Tab` to achieve the effect you described, you can try remapping `Tab` to `Down`.

**Reply by jrock2004** [reply]
Trying to find that option. I will keep searching

**Reply by iamxiaojianzheng** [reply, marked as answer]
It worked for me

```lua
return {
  "ibhagwan/fzf-lua",
  opts = function(_, opts)
    return {
      keymap = {
        fzf = {
          ["tab"] = "down",
          ["shift-tab"] = "up",
        },
      },
    }
  end,
}
```
